### DISCUSSION TITLE
lru is not a constructor

### DISCUSSION THREAD

**Discussion by NPCBOSS** [nuxt/nuxt#24495] [Questions]
I use mysql2 in nuxt3，My local development environment is running normally, but after using Docker to package, an error will be reported when calling the interface，the pages is normal

**lru is not a constructor**

lru-cache is mysql dependencies

![image](https://github.com/nuxt/nuxt/assets/38394370/2b7f59d2-b89d-4324-b36a-6f2e460324a2)
![image](https://github.com/nuxt/nuxt/assets/38394370/d3e9d6cb-a14c-49ff-ac2a-80a0c6f17559)

**Comment by NPCBOSS** [comment]
mysql2  version is 3.6.5，just use simple sql 

eg

`
 const querySql = `SELECT * FROM table where mobile = '${phone}';`
  const [rows] = await pool.query <any[]>(querySql)
`

**Comment by JasonFeng03** [comment]
This is because the lru-cache on which nuxt builds output mysql2 is a symbolic link.  Link failed while moving file.  You can go in and see that the folder is empty.  I'm looking for a solution, too
